### ISSUE TITLE
First training step, no loss computed for training set.

### ISSUE THREAD

**Issue by gergf** [tflearn/tflearn#695]
```
--
Training Step: 1  | time: 20.876s
| Adam | epoch: 001 | loss: 0.00000 - acc: 0.0000 | val_loss: 0.55601 - val_acc: 0.1396 -- iter: 8/8
--
```

I don't understand why in the first training step the loss is not computed. Any particular reason?

Thank you.

**Comment by aymericdamien** [comment]
Your training is likely diverging. It happened that when your training has 'nan', it output 0 loss.

**Comment by gergf** [comment]
Hmm. It's weird because there is no problems in further iterations... 
That makes me think that maybe I have some problem with the initialization of the weights.

I'll change it and post if that solve the problem.

**Comment by JoaoCarabetta** [comment]
@gergf Have you solved it?
